installing/installing_rhv/installing-rhv-customizations.adoc

```diff
@@ -7,9 +7,13 @@ toc::[]
 
 [WARNING]
 ====
-Due to a known issue with installing {product-title} versions 4.4 and 4.5 on {rh-virtualization-first} 4.4.1, you must customize `install-config.yaml` as described in link:https://access.redhat.com/solutions/5374251[Openshift IPI installation on RHV-4.x failed with "Error: timeout while waiting for state to become 'up' (last state: 'down', timeout: 10m0s)"]. This defect is fixed in {rh-virtualization} 4.4.2.
+Due to a known issue with installing {product-title} versions 4.4 and 4.5 on {rh-virtualization-first} 4.4.1, you must customize `install-config.yaml` as described in link:https://access.redhat.com/solutions/5374251[Openshift IPI installation on {rh-virtualization}-4.x failed with "Error: timeout while waiting for state to become 'up' (last state: 'down', timeout: 10m0s)"]. This defect is fixed in {rh-virtualization} 4.4.2.
```

Comment:
Would it also make sense to change 'OpenShift' to `{product-title}` here? The same comment goes for other instances of this warning in this PR.

Reply:
Actually please disregard this suggestion. This would render the link title to be different from the title on the page we are linking to.